src/dependency-manager/src/component-config/base.ts

```diff
@@ -43,12 +44,14 @@ export abstract class ComponentConfig extends ConfigSpec {
     return `${this.getRef()}-interfaces`;
   }
 
-  getComponentVersion() {
-    return this.getRef().split(':')[1];
+  //TODO:87:why are these implemented in base.ts instead of v1.ts
```

Comment:
I noticed a couple of these methods are implemented here in the `base.ts` instead of `v1.ts`. Any particular reason?